src/netaddress.cpp

```diff
@@ -674,7 +674,7 @@ bool CNetAddr::GetInAddr(struct in_addr* pipv4Addr) const
  */
 bool CNetAddr::GetIn6Addr(struct in6_addr* pipv6Addr) const
```

Comment:
In commit 7b4005008bce3b2d77a4a40db7db7f6626170f87, do you want to keep this function name and parameter?

Reply:
The function semantics remain unchanged - it is still used to "get `struct in6_addr`", so I think it is ok to leave it as is.